webpack.config.ts

```diff
@@ -3,6 +3,7 @@
 import { Configuration as WebpackConfiguration } from 'webpack';
 import { Configuration as WebpackDevServerConfiguration } from 'webpack-dev-server';
 import * as path from 'path';
+//import fs from 'fs';
```

Comment:
Was it intentional to leave the comment in for future reference?

Reply:
Nope...i will update the PR by removing the comment lines.